### PR TITLE
Small UI refinements

### LINKED ISSUE #2305: Move alignment options above color presets
Reasoning: I found myself using these options quite a bit for stories with image layouts and having the alignment options shift on me was disorientating.

### LINKED ISSUE #2616: Hide elements tab behind flag
## Feature Description

Let's put this behind a flag until it's implemented.

## Additional Context

Screen Shot 2020-06-19 at 7 43 05 AM

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### LINKED ISSUE #2619: Editor UI refinements for beta
## Feature Description

Move "Nothing selected" to the center and a bit further down
Design & Document tabs should be aligned to the left

DesignPanel - Page (8)

Remove white lines at the bottom:

Screen Shot 2020-06-19 at 7 52 26 AM

Screen Shot 2020-06-19 at 7 51 52 AM

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

* ~Hide elements tab behind flag~ -> now done in #2658 / #2616
* Some small UI refinements as per Sam's feedback.
* Disable resizing and collapsing presets panel when there are no presets
* Move alignment options above color presets
* Remove whitepace between drag handle and color swatches
* Expand presets panel when entering edit mode
* Improve grid layout of color swatches 

## User-facing changes

Screenshot 2020-06-19 at 17 19 52

Screenshot 2020-06-22 at 13 52 11

Screenshot 2020-06-23 at 21 17 31

## Testing Instructions

* ~Verify that elements tab is no longer visible.~ -> now done in #2658 / #2616
* Verify that alignment options are above color presets
* Verify that there is no whitepace between thee presets color swatches and the bottom border
* Verify that a collapsed presets panel is expanded when entering edit mode

---

Fixes #2619
Fixes #2305
See #2616

### CODE REVIEW

**Comment by swissspidy** [comment]
Another small to-do:

Screen Shot 2020-06-19 at 11 13 51 AM

**Comment by spacedmonkey** [comment]
> Another small to-do:
> 
> Screen Shot 2020-06-19 at 11 13 51 AM

Screenshot 2020-06-22 at 11 50 22
The UX looks very different in this branch. Can you explain in more detail @swissspidy what was changed and why?

**Comment by swissspidy** [comment]
> The UX looks very different in this branch. Can you explain in more detail @swissspidy what was changed and why?

The first screenshot is the desired end state. This PR is a WIP towards that desired end state. A summary of changes is in the PR description

File: assets/src/edit-story/components/panels/panel/shared/title.js [hunk 1]
```diff
@@ -159,6 +159,7 @@ function Title({
           handleHeightChange={handleHeightChange}
           handleExpandToHeightChange={handleExpandToHeightChange}
           handleDoubleClick={resetHeight}
+          position="top"
```
Comment (miina) [inline assets/src/edit-story/components/panels/panel/shared/title.js]: Any reason to add this if `top` is the default anyway?

**Comment by miina** [comment]
From testing:

I can still see the collapse and edit icon even if there are no presets:
Screenshot 2020-06-22 at 15 05 14

Could you mention which [points from here](https://github.com/google/web-stories-wp/pull/2620#issuecomment-647360677) are addressed -- is it just the 4 currently?
Currently seeing this (there's the gap on the right still):
Screenshot 2020-06-22 at 15 07 38

File: assets/src/edit-story/components/panels/panel/shared/title.js (hunk 1 above)
Comment (swissspidy) [reply assets/src/edit-story/components/panels/panel/shared/title.js]: No, can probably remove it.

**Comment by swissspidy** [comment]
> I can still see the collapse and edit icon even if there are no presets:

Checking...Actually didn't see those locally before.

> Could you mention which points from here are addressed -- is it just the 4 currently?
> Currently seeing this (there's the gap on the right still):

Only 1 & 4. I'll leave  2 & 3 for  #2586 as it didn't seem trivial.

**Comment by swissspidy** [comment]
> I can still see the collapse and edit icon even if there are no presets:

Cannot reproduce this. If there are no presets it looks like this:

Screenshot 2020-06-22 at 16 11 57

**Comment by miina** [comment]
Hmm, I also can't reproduce it anymore, odd. 🤷

**Review by miina — APPROVED** [review]
LGTM, just the one nit about using the already existing default value.

**Pull request merged**